addons/spreadsheet/static/src/o_spreadsheet/o_spreadsheet.js

```diff
@@ -31079,7 +31079,7 @@ function getCopyMenuItem(figureId, env) {
             env.model.dispatch("COPY");
             await env.clipboard.write(env.model.getters.getClipboardContent());
         },
-        icon: "o-spreadsheet-Icon.COPY",
+        icon: "o-spreadsheet-Icon.CLIPBOARD",
```

Comment:
This file is generated by [o-spreadsheet](https://github.com/odoo/o-spreadsheet) build tools :wink: it should not be edited because the changes would be overwritten the next time we update the lib (~once a week).

We can do the changes our-self or you can also open a PR on o-spreadsheet repo.

Same thing for `o_spreadsheet.xml`

Reply:
@LucasLefevre The changes have been removed here, and a [pull request](https://github.com/odoo/o-spreadsheet/pull/5192) has been created in the `o-spreadsheet` repository to apply these updates directly. Let me know if any further adjustments are needed.